hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -1815,7 +1814,7 @@ public List<ExpiredMultipartUploadsBucket> getExpiredMultipartUploads(
              mpuInfoTableIterator = getMultipartInfoTable().iterator()) {
 
       final long expiredCreationTimestamp =
-          Instant.now().minus(expireThreshold).toEpochMilli();
+          expireThreshold.negated().plusMillis(Time.now()).toMillis();
```

Comment:
https://issues.apache.org/jira/browse/HDDS-7764 lists going to Time.now() as a temporary solution. We should switch to UTC time and not be dependent on local time zone configuration.

Reply:
Thank you for the review. I agree with switching from `Time.now()` to `Instant.now()` (which will be addressed in HDDS-7911). However, currently the issue is that `OmKeyInfo` `creationTimestamp` and `modificationTimestamp`  already using `Time.now()`. If we switch to `Instant.now()` instead, the timestamps in the key / file table will be a mix of UTC and local time zone.